scripts/nbd_client_manager.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+
+"""
+Provides functions and a CLI for safely connecting to and disconnecting from
+NBD devices.
+"""
+
+import argparse
+import logging
+import os
+import subprocess
+import time
+import fcntl
+
+
+LOGGER = logging.getLogger("nbd_client_manager")
```

Comment:
I think this logs at `warning` level by default

Reply:
I think so, good spot, I'll increase the logging level to `debug`.

Reply:
:+1: There is a `LOGGER.setLogLevel()` call for that

Reply:
I think we need to do more configuration to redirect log messages into syslog using `SysLogHandler`.